api/src/main/java/com/example/api/dto/response/ranking/RankingEntry.java

```diff
@@ -0,0 +1,24 @@
+package com.example.api.dto.response.ranking;
+
+import com.example.api.model.user.HeroType;
+import com.example.api.model.user.User;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class RankingEntry {
```

Comment:
Dzisiaj robiąc taska ogarnąłem, że w większości tych dto/response są w postaci:
*nazwa*Response
Także, żeby to było konsekwentne to imo lepiej żeby wszystkie były tak zapisane (można by od razu wszystkie niepasujące zmienić)